### PR TITLE
Trace decode time of LR_WasmDecoder

### PR DESCRIPTION
**Pull request by jing-bao**
Adding trace of LR_WasmDecoder decode time.
Also **need to add to v8 repo** FLAG_trace_wasm_decode_time at the same time.

### CODE REVIEW

**Comment by jfbastien** [comment]
@jing-bao are you a member of the W3C community group, as per the [contributing](https://github.com/WebAssembly/design/blob/master/Contributing.md) document?

**Comment by jing-bao** [comment]
Nope, I'm working to join the group now, thanks for reminding.
The check fails because FLAG_trace_wasm_decode_time isn't in v8 now, which I'll push to v8 if you think this patch is OK.
Do you have any comments on this patch? @jfbastien @titzer

File: src/wasm/decoder.cc [hunk 1]
```diff
@@ -156,6 +161,10 @@ class LR_WasmDecoder : public Decoder {
       }
     }
     
+    if (FLAG_trace_wasm_decode_time && ok()) {
```
Comment (titzer) [inline src/wasm/decoder.cc]: You can combine this if with the following if, nesting the decode tracing inside.

Comment (jing-bao) [inline src/wasm/decoder.cc]: done.

**Comment by titzer** [comment]
This mostly looks good. Spin up a v8 CL to add the appropriate flag.

**Comment by jing-bao** [comment]
Close this PR because weilianglin opened a new one.

**Pull request closed without merging**